src/wasm.h

```diff
@@ -91,6 +91,11 @@ struct Name : public cashew::IString {
 // An index in a wasm module
 typedef uint32_t Index;
 
+inline Index toIndex(size_t index, size_t max) {
```

Comment:
I'm not sure about the second parameter. It does help in some cases, but we may not always have that information around, and we'd need to write a lot of `getFunction()->getNumLocals()`. I suggest just asserting on just the size of the type, `assert(index <= std::numeric_limits<uint32_t>::max())`.